### PR TITLE
initial latex framework

### PR DESCRIPTION
**Pull request by jphall663**
setup latex sections

add authors back onto large mli deck w/ notation

### CODE REVIEW

File: tex/interpretable_ml.tex [hunk 1]
```diff
@@ -54,6 +54,8 @@
 
 \title{\large \bfseries Interpretable Machine Learning}
 
+\author{Wen Phan, Lauren DiPerna, Patrick Hall\\[3ex]}
+
```
Comment (navdeep-G) [inline tex/interpretable_ml.tex]: Can you please add me as an author to the end of this? I did edits locally before the initial push with no authors listed.

Comment (jphall663) [reply tex/interpretable_ml.tex]: I will add you as author and sections for PDP and ICE.

Comment (navdeep-G) [reply tex/interpretable_ml.tex]: Sections for PDP and ICE? Those should be there in this text.

Comment (jphall663) [reply tex/interpretable_ml.tex]: Yep - they are there now - merging. 

Addresses #14

**Pull request merged**